boardwithleds.h

```diff
@@ -0,0 +1,29 @@
+#ifndef BOARDWITHLEDS_H
+#define BOARDWITHLEDS_H
+
+#include <QSerialPort>
+#include <QString>
+#include "ledinfo.h"
+
+class BoardWithLeds
+{
+public:
+    BoardWithLeds(Led led);

```

Comment:
If your constructor takes only one argument it is better to mark it `explicit` in order to avoid possibility of implicit type cast